### ISSUE TITLE
LDAP Support

### ISSUE THREAD

**Issue by ryansnowden** [fiduswriter/fiduswriter#47]
A lot of campuses use LDAP for auth and profile information. I think LDAP auth would be excellent way to integrate.

**Comment by stonfute** [comment]
It's not really complicated to implement ldap auth on Django. The main problem is to deal with model used by classic user auth. I will check how it has been done with social auth, i will try to make the same with LDAP.

**Comment by dalareo** [comment]
Any new on this feature? We are integrating Fiduswriter in educaaas.io and LDAP would be great!!

**Comment by johanneswilm** [comment]
Not really. We opted for using our own homemade login integration in the OJS integration. But we would of course merge it if someone else woudl add this.

**Comment by dalareo** [comment]
What a pity. I guess integrating LDAP is not that hard but I am not able myself. Let's see if someone joins an ad-hoc team and we are able to move this forward. Having LDAP would be very nice in a corporative (universities) environment

**Comment by andrewwippler** [comment]
LDAP can be implemented through a [Dex Connector](https://dexidp.io/docs/connectors/ldap/). Then, one can use the [OpenID connect authenticator to point at the Dex Idp](https://django-allauth.readthedocs.io/en/latest/socialaccount/providers/openid_connect.html) in Fiduswriter's django-allauth.
